### PR TITLE
Orange3 single cell

### PR DESCRIPTION
**Pull request by JakaKokosar**

### CODE REVIEW

**Comment by JakaKokosar** [comment]
@marcelotrevisani Can you please look at this?

File: recipes/Orange3-SingleCell/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+    
+requirements:
+  build:
```
Comment (marcelotrevisani) [inline recipes/Orange3-SingleCell/meta.yaml]: Please change that to ``host``

File: recipes/Orange3-SingleCell/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py2k]
```
Comment (marcelotrevisani) [inline recipes/Orange3-SingleCell/meta.yaml]: Is this project a pure Python project?
If it is you can add ``noarch::python``, remove the ``  skip: True  # [py2k]`` and in ``requirements`` you have to put ``- python >2``

File: recipes/Orange3-SingleCell/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```
Comment (marcelotrevisani) [inline recipes/Orange3-SingleCell/meta.yaml]: It seems that you have some entry points, so you may add them
Please see
https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

**Comment by bgruening** [comment]
@marcelotrevisani thanks for the ping.

@JakaKokosar indeed there is a sub-conda community, depending on conda-forge that is dedicated to Bioinformatics. If you like please submit your pkg to [Bioconda](http://bioconda.github.io/)

**Comment by JakaKokosar** [comment]
@bgruening @marcelotrevisani 

Please read this regarding bioconda:
https://github.com/conda-forge/staged-recipes/pull/5184#issuecomment-372409897

**Comment by bgruening** [comment]
I see. Thanks for the clarification :)

**Comment by JakaKokosar** [comment]
@marcelotrevisani, is this OK now?

**Comment by marcelotrevisani** [comment]
LGTM! I just wrote wrong one thing to you but I fixed, to skip python 2.7 just put ``- python >=3`` and should be good, I wrote that wrong, sorry! But I already fixed and as soon it passed I will merge it

Thanks for adding this to ``conda-forge`` ! :tada:

**Pull request merged**